docs/detections/api/rules/rules-api-update.asciidoc

```diff
@@ -60,6 +60,7 @@ occurred
 |type |String a|Data type on which the rule is based:
 
 * `eql`: EQL query (see {ref}/eql.html[Event Query Language]).
+* `esql`: ES\|QL query (see {ref}/esql.html[Elasticsearch Query Language]).
```

Comment:
```suggestion
* `esql`: ES|QL query (refer to {ref}/esql.html[Elasticsearch Query Language]).
```

Reply:
Ah, ok. Yeah, if it's not rendering in the table row, it'll need to be escaped. Feel free to reject and resolve this edit!

Reply:
Here's the edit without removing the escape:

```suggestion
* `esql`: ES\|QL query (refer to {ref}/esql.html[Elasticsearch Query Language]).
```

Reply:
I changed "see" to "refer" because we try to avoid ableist langugage in our docs wherever possible. Here's our style guidelines for reference: https://docs.elastic.dev/content-design/inclusivity#avoid-violent-offensive-ableist-terminology